caiman/source_extraction/volpy/volpy_gui.py

```diff
@@ -15,18 +22,15 @@
 from pyqtgraph import FileDialog
 from pyqtgraph.Qt import QtGui
 from pyqtgraph.parametertree import Parameter, ParameterTree
-import PyQt5
-from PyQt5 import QtWidgets
-from PyQt5.QtWidgets import QApplication, QShortcut
 import random
 from skimage.draw import polygon
 import sys
 
 import caiman as cm
 from caiman.external.cell_magic_wand import cell_magic_wand_single_point
-    
+
 os.environ["QT_QPA_PLATFORM_PLUGIN_PATH"] = os.fspath(
-    Path(PyQt5.__file__).resolve().parent / "Qt5" / "plugins")
+    Path(PySide6.__file__).resolve().parent / "Qt5" / "plugins")
```

Comment:
I wonder if this would still be valid with `Qt5` in the path when using pyside6?